bundles/org.openhab.binding.pentair/README.md

```diff
@@ -72,133 +87,200 @@ The following table shows the available configuration parameters for each thing.
+| "                                 | solartemperature | Number:Temperature | R | Solar temperature sensor reading. |
+| "                                 | airtemperature   | Number:Temperature | R | Air temperature sensor reading. |
+| "                                 | uom              | String             | R | Unit of measure. Values: CELCIUS, FARENHEIT. |
+| "                                 | servicemode      | Switch             | R | Indicates whether controller is in service mode. |
+| "                                 | solaron          | Switch             | R | Indicates whether solar heat is on. |
+| "                                 | heateron         | Switch             | R | Indicates whether heater is on. |
+
+#### Working with schedules
+
+This binding allows both reading and writing of schedules and supports up to 9 schedules. Programming of a schedule can be accomplished either by using the discrete channels linked to items (i.e. type, start, end, circuit, days) or you can concatenate those and use the `schedule` channel saved as a comma delimited string.  To prevent erroneous writes to the schedules though, one must write to the `type` channel the same value twice within 5 sec.
+
+### Thing: IntelliChlor
+
+Represents an Intellichlor module connected in your system.  Currently, the values here are readonly.
+
+| Channel              | Type       |     | Description |
+| :------------------: | :----:     | :-: | :---------- |
+| saltoutput           | Number     | R   | Current salt output %. |
+| salinity             | Number     | R   | Salinity (ppm). |
+| ok                   | Switch     | R   | System is operating normally. |
+| lowflow              | Switch     | R   | Water flow rate is low. |
+| lowsalt              | Switch     | R   | Low salt level. |
+| verylowsalt          | Switch     | R   | Very low salt level. |
+| highcurrent          | Switch     | R   | High current level. |
+| cleancell            | Switch     | R   | Clean cell. |
+| lowvoltage           | Switch     | R   | Low voltage. |
+| lowwatertemp         | Switch     | R   | Water temperature is too low for chlorine generation. |
+| commerror            | Switch     | R   | Communication error. |
+
+### Thing: IntelliFlo
+
+Represents and interfaces to an Intelliflo pump.  When a controller is active in the system all pump values are read only since the pump can only have one master at a time.  If no controller is present or the controller is in service mode, the pump can be controlled directly from OpenHab.
+
+| Channel              | Type       |     | Description |
+| :------------------: | :----:     | :-: | :---------- |
+| run                  | Switch    | RW | Indicates whether the pump is running. |
+| rpm                  | Number    | RW  | Pump RPM |
+| gpm                  | Number    | R  | Pump GPM (only valid for VF pumps) |
```

Comment:
This could be `Number:VolumetricFlowRate`. See https://www.openhab.org/docs/concepts/units-of-measurement.html#list-of-units